marathon/util.py

```diff
@@ -9,6 +9,10 @@
 import re
 
 
+def is_stringy(obj):
+    return isinstance(obj, str) or isinstance(obj, unicode)
```

Comment:
This is equivalent to the previous code, `isinstance(obj, types.StringTypes)`. I don't know how well this will work in python 3, where unicode becomes str and str becomes bytes.

If this library is officially targeting python 3, can we add it to tox.ini?